### DISCUSSION TITLE
Daily 9/11

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [rpgrca/proyecto-ispc-fullstack#284] [General]
Ayer estuve estudiando Inglés. También agregué los enlaces a todas la páginas y subí todo al repositorio.
Hoy estaré estudiando Inglés y empezando con  el repaso de SGC para la próxima evaluación.

**Comment by rpgrca** [comment]
Ayer no pude hacer nada porque estuve con mucho trabajo, así que lo que iba a hacer ayer lo hago hoy. También me pongo a editar los videos para armar el video final.

**Comment by rocappellano** [comment]
Estudié inglés, y ahora me dedícaréa preparar el vídeo final

**Comment by Brian-Almada** [comment]
Ayer estuve con inglés. Hoy preparé el portfolio para la pág de Almoneda.
